caesars-palace/blocks/columns/columns.js

```diff
@@ -1,4 +1,15 @@
 export default function decorate(block) {
   const cols = [...block.firstElementChild.children];
   block.classList.add(`columns-${cols.length}-cols`);
+
+  if (block.classList.contains('cet-card') && cols.length === 2) {
+    // image placement
+    if (cols[0].querySelector('picture')) {
```

Comment:
Is there always a picture? Because if so, this could become an if/else statement instead of using querySelector twice

Reply:
There doesn't have to be an image in the column in which case no additional classes would be added.